src/app/containers/Preferences/Preferences.jsx

```diff
@@ -570,36 +577,9 @@ class PreferencesPage extends PureComponent {
                 return defaultColour;
             },
             getCurrentColor: (part, defaultColour) => {
-                const { visualizer } = this.state;
-                if (visualizer.theme === LIGHT_THEME) {
-                    return LIGHT_THEME_VALUES.backgroundColor;
-                } else if (visualizer.theme === DARK_THEME) {
-                    return DARK_THEME_VALUES.backgroundColor;
-                } else {
-                    return this.visualizerConfig.get(visualizer.theme + ' ' + part)
-                        ? this.visualizerConfig.get(visualizer.theme + ' ' + part)
-                        : defaultColour;
-                }
-            },
-            resetCustomThemeColours: (theme) => {
-                let themeColours = DARK_THEME_VALUES;
-                if (theme === CUST_LIGHT_THEME) {
-                    themeColours = LIGHT_THEME_VALUES;
-                }
-                this.visualizerConfig.set(theme + ' Background', themeColours.backgroundColor);
-                this.visualizerConfig.set(theme + ' Grid', themeColours.gridColor);
-                this.visualizerConfig.set(theme + ' X Axis', themeColours.xAxisColor);
-                this.visualizerConfig.set(theme + ' Y Axis', themeColours.yAxisColor);
-                this.visualizerConfig.set(theme + ' Z Axis', themeColours.zAxisColor);
-                this.visualizerConfig.set(theme + ' Limit', themeColours.limitColor);
-                this.visualizerConfig.set(theme + ' Cutting Coordinate Lines', themeColours.cuttingCoordinateLines);
-                this.visualizerConfig.set(theme + ' Jogging Coordinate Lines', themeColours.joggingCoordinateLines);
-                this.visualizerConfig.set(theme + ' G0', themeColours.G0Color);
-                this.visualizerConfig.set(theme + ' G1', themeColours.G1Color);
-                this.visualizerConfig.set(theme + ' G2', themeColours.G2Color);
-                this.visualizerConfig.set(theme + ' G3', themeColours.G3Color);
-
-                pubsub.publish('theme:change');
+                return this.visualizerConfig.get(CUST_THEME + ' ' + part)
+                    ? this.visualizerConfig.get(CUST_THEME + ' ' + part)
+                    : defaultColour;
```

Comment:
The .get function actually takes a second argument that is a default value if it's not able to find the config from the first argument string you provided. It's a little cleaner to use that instead of the ternary operator you have here.